libbeat/template/load.go

```diff
@@ -156,6 +157,15 @@ func (l *ESLoader) Load(config TemplateConfig, info beat.Info, fields []byte, mi
 // template if it exists. If you wish to not overwrite an existing template
 // then use CheckTemplate prior to calling this method.
 func (l *ESLoader) loadTemplate(templateName string, template map[string]interface{}) error {
+	sameTemplate, sameTemplateErr := l.sameAsExistingTemplate(templateName, template)
+	if sameTemplateErr != nil {
+		l.log.Infof("Get template %s from Elasticsearch failed. Template check will be ignored.", templateName)
```

Comment:
```suggestion
		l.log.Infof("Get template %s from Elasticsearch failed. Template check will be ignored: %s", templateName, sameTemplateErr)
```

Reply:
might also want to be `Warnf` instead of `Infof`